tracee-ebpf/tracee/argprinters.go

```diff
@@ -53,17 +55,15 @@ func (t *Tracee) prepareArgs(ctx *context, args map[string]interface{}) error {
 		}
 	}
 
-	if ctx.EventID == MemProtAlertEventID {
-		if alert, isAlert := args["alert"].(alert); isAlert {
-			args["alert"] = PrintAlert(alert)
-		}
-	}
-
 	// EventIDs which are optionally parsed/enriched into human readable formats
 	if !t.config.Output.ParseArguments {
 		return nil
 	}
 	switch ctx.EventID {
+	case MemProtAlertEventID:
+		if message, isMessage := args["alert"].(external.Message); isMessage {
```

Comment:
message?

Reply:
No. "alert" is the name of the argument, and I think we should keep this name